app-backend/api/src/main/scala/exports/Routes.scala

```diff
@@ -78,10 +80,15 @@ trait ExportRoutes extends Authentication
   def createExport: Route = authenticate { user =>
     entity(as[Export.Create]) { newExport =>
       authorize(user.isInRootOrSameOrganizationAs(newExport)) {
-        onSuccess(write[Export](Exports.insertExport(newExport, user))) { export =>
-          kickoffProjectExport(export.id)
-          complete(export)
-        }
+        (for {
+          ei <- newExport.exportOptions.as[ExportOptions].toOption
+        } yield {
+          onSuccess(write[Export](Exports.insertExport(newExport, user))) { export =>
+            kickoffProjectExport(export.id)
+            complete(export)
+          }
+        }).getOrElse(
+          complete((StatusCodes.BadRequest, "Parameter 'exportOptions' could not be decoded to type 'ExportOptions'")))
```

Comment:
It could read something like, `Parameter 'exportOptions' is invalid`, but that seems less clear. I'm not sure if there's an easy way to get what *exactly* the decoder choked on, so if there isn't then this is about as specific as it can be. My thinking was that this at least makes it clear that the JSON itself is valid, but the shape is wrong.